packages/core/src/app/cart/Redeemable.tsx

```diff
@@ -136,7 +136,7 @@ const RedeemableForm: FunctionComponent<
                 return <TranslatedString id="redeemable.coupon_location_error" />;
 
             default:
-                return <TranslatedString id="redeemable.code_invalid_error" />;
+                return errorMessage?.trim() || <TranslatedString id="redeemable.code_invalid_error" />;
```

Comment:
`errorMessage?.trim()` Do we need to trim the message?

Reply:
I think we better do it, because like I mentioned, " " will give you
![image](https://github.com/user-attachments/assets/736c2cdb-3761-4ea4-bb88-cd8a8cecd612)

we need to use fallback value instead (even through it's not a usual response, but for the robust)